CMR/python/README.md

```diff
@@ -17,7 +17,26 @@ A wrapper for [github.com/nasa/Common-Metadata-Repository][git_cmr] in python
 
 ## Building and Installing
 
-As of this time the CMR library is not hosted on [pypy.org][pypi]. To install the library from a [wheel][wheel] file it will need to be generated locally using the 'runme.sh' script. On a command line, do the following from the python/CMR directory:
+As of this time the CMR library is not hosted on [pypy.org][pypi]. There are several ways to install the library:
```

Comment:
Grammar fix
"As of this time the CMR library is not hosted" => "This tool is not presently hosted..."